oak_functions/examples/weather_lookup/example.toml

```diff
@@ -16,6 +16,5 @@ additional_args = [
 rust = { Cargo = { cargo_manifest = "oak_functions/client/rust/Cargo.toml" }, additional_args = [
   "--uri=http://localhost:8080",
   "--request={\"lat\":52,\"lon\":0}",
-  "--expected-response={\"temperature_degrees_celsius\":16}"
```

Comment:
Not part of this PR, but we should figure out a better way for testing this. The java client still checks for the expected value, and it changes every time the lookup data is regenerated.